### PR TITLE
fix submenus

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

before / after

https://github.com/rainbow-me/browser-extension/assets/6232729/a26ea187-fb58-47e1-af31-e750dd15d66e

https://github.com/rainbow-me/browser-extension/assets/6232729/c4c7f0c5-69f6-43cd-a9c6-34257a7897d8

### CODE REVIEW

**Review by estebanmino — CHANGES_REQUESTED** [review]
the submenu is centered but it shouldn’t scale down, it should maintain the same width than before the animation

EDIT: this is not an issue anymore

File: src/entries/popup/components/DropdownMenu/DropdownSubMenu.tsx [hunk 1]
```diff
@@ -176,16 +176,6 @@ export const DropdownSubMenu = ({
       <DropdownMenuTrigger asChild>
         <Box ref={subMenuElementRef}>{subMenuElement}</Box>
       </DropdownMenuTrigger>
-      <DropdownMenuContent
```
Comment (estebanmino) [inline src/entries/popup/components/DropdownMenu/DropdownSubMenu.tsx:179]: this shouldn't be removed

Comment (greg-schrammel) [reply src/entries/popup/components/DropdownMenu/DropdownSubMenu.tsx:179]: what is this for?

**Review by estebanmino — APPROVED** [review]
🪨

**Review by derHowie — APPROVED** [review]

**Pull request merged**